web/src/components/tasks/anytime-action/AnytimeActionLicenseRenewalElement.test.tsx

```diff
@@ -0,0 +1,179 @@
+    ];
+    const secondLineAddress = licenseAddress.addressLine2 ? ` ${licenseAddress.addressLine2}` : "";
+    const address =
+      `${licenseAddress.addressLine1}${secondLineAddress}, ${licenseAddress.zipCode} NJ`.toUpperCase();
+    renderAnytimeActionLicenseRenewalElement(anytimeActionLicense[0]);
+    expect(screen.getByText(licenseAddress.name.toUpperCase())).toBeInTheDocument();
+    expect(screen.getByText(address)).toBeInTheDocument();
+  });
+
+  it("renders Anytime Action License RenewalElement with last updated date", () => {
+    const licenseName = randomElementFromArray(Object.values(taskIdLicenseNameMapping));
+    const licenseAddress = {
+      name: "Business Sample",
+      addressLine1: "1 Business Way",
+      addressLine2: "Suite 10",
+      zipCode: "08211",
+    };
+
+    useMockBusiness({
+      licenseData: generateLicenseData({
+        licenses: {
+          [licenseName]: generateLicenseDetails({
+            licenseStatus: "ACTIVE",
+            nameAndAddress: {
+              ...licenseAddress,
+            },
+            lastUpdatedISO: "2018-04-23T10:26:00.996Z",
+          }),
+        },
+      }),
+    });
+
+    const anytimeActionLicense = [
+      generateAnytimeActionLicenseRenewal({
+        name: "some-license-name",
+        urlSlug: "some-url",
+        filename: "some-filename-license",
+        licenseName,
+      }),
+    ];
+
+    renderAnytimeActionLicenseRenewalElement(anytimeActionLicense[0]);
+    expect(
+      screen.getByText(
+        parseDateWithFormat("2018-04-23T10:26:00.996Z", defaultDateFormat).format(licenseSearchDateFormat)
+      )
+    ).toBeInTheDocument();
+  });
+
+  it("renders Anytime Action License Renewal Element with checklist items", () => {
+    const licenseName = randomElementFromArray(Object.values(taskIdLicenseNameMapping));
+    const licenseAddress = {
+      name: "Business Sample",
+      addressLine1: "1 Business Way",
+      addressLine2: "Suite 10",
+      zipCode: "08211",
+    };
+
+    useMockBusiness({
+      licenseData: generateLicenseData({
+        licenses: {
+          [licenseName]: generateLicenseDetails({
+            licenseStatus: "ACTIVE",
+            nameAndAddress: {
+              ...licenseAddress,
+            },
+            checklistItems: [{ title: `checklist-title-1`, status: "ACTIVE" }],
+            lastUpdatedISO: "2018-04-23T10:26:00.996Z",
+          }),
+        },
+      }),
+    });
+
+    const anytimeActionLicense = [
+      generateAnytimeActionLicenseRenewal({
+        name: "some-license-name",
+        urlSlug: "some-url",
+        filename: "some-filename-license",
+        licenseName,
+      }),
+    ];
+
+    renderAnytimeActionLicenseRenewalElement(anytimeActionLicense[0]);
+    const applicationCheckListItems = screen.getByText(
+      Config.licenseSearchTask.applicationChecklistItemsText
+    );
+
+    expect(applicationCheckListItems).toBeInTheDocument();
+    fireEvent.click(applicationCheckListItems);
```

Comment:
[sand] Hey just a heads up, we should probably be using `userEvent` here since [it more realistically simulates user interactions](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent). I believe `fireEvent` should just be used as a last resort when an event is not covered by `userEvent`.